cmd/2019_3/main.go

```diff
@@ -73,18 +61,18 @@ func (g *grid) addWire(wirePath string, index int) {
 
 		switch direction[0] {
 		case 'R': // Right
-			currentStep = g.add(&currentPt, &currentPt.X, step, currentStep, index)
+			currentStep = g.add(currentPt, &currentPt.X, step, currentStep, index)
 		case 'L': // Left
-			currentStep = g.add(&currentPt, &currentPt.X, -step, currentStep, index)
+			currentStep = g.add(currentPt, &currentPt.X, -step, currentStep, index)
 		case 'U': // Up
-			currentStep = g.add(&currentPt, &currentPt.Y, step, currentStep, index)
+			currentStep = g.add(currentPt, &currentPt.Y, step, currentStep, index)
 		case 'D': // Down
-			currentStep = g.add(&currentPt, &currentPt.Y, -step, currentStep, index)
+			currentStep = g.add(currentPt, &currentPt.Y, -step, currentStep, index)
 		}
 	}
 }
 
-func (g *grid) add(position *point, coordinate *int, step, currentStep, index int) int {
+func (g *grid) add(position *point.Point, coordinate *int, step, currentStep, index int) int {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [maximum number of arguments per function exceeded; max 4 but got 5](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498955)